<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="title">
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Prima di iniziare</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="text">
     <property name="styleSheet">
      <string notr="true">font: 17pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Rubik Light</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="text2">
     <property name="styleSheet">
      <string notr="true">font: 17pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>text2</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="audio_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Rubik Light&quot;;
background-color: rgb(250, 250, 250);</string>
       </property>
       <property name="text">
        <string>Prova audio</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="audio_icon">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="text3">
     <property name="styleSheet">
      <string notr="true">font: 17pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>text3</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="done_button">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(0, 64, 130);
font: 14pt &quot;Rubik Light&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>Fatto</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
